charts/quickwit/values.yaml

```diff
@@ -116,28 +125,34 @@ indexer:
 
   # Extra env for indexer
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
   extraEnvFrom: []
     # - secretRef:
     #     name: quickwit-indexer
     # - configMapRef:
-    #     name: quickwit-indexer
+  #     name: quickwit-indexer
+
+  # extraVolumes -- Additional Volumes to use with Vector Pods.
+  extraVolumes: []
+
+  # extraVolumeMounts -- Additional Volume to mount into Vector Containers.
+  extraVolumeMounts: []
 
   resources: {}
     # limits:
     #   cpu: 100m
     #   memory: 128Mi
     # requests:
     #   cpu: 100m
-    #   memory: 128Mi
+  #   memory: 128Mi
 
   ## Pod distruption budget
   podDisruptionBudget: {}
     # maxUnavailable: 1
-    # minAvailable: 2
+  # minAvailable: 2
 
   updateStrategy: {}
-    # type: RollingUpdate
+  # type: RollingUpdate
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed